README.md

```diff
@@ -2,18 +2,21 @@
 
 ## Getting Started
 
-A virtual machine is used to encapsulate Docker dependencies. `docker-compose` is used within the VM to manage running the application and developing against it.
-
 ### Requirements
 
-- Vagrant 1.8+
-- VirtualBox 5.0+
-- Ansible 1.8+ (on host)
 - AWS CLI 1.10+
 - AWS Account (to store artifacts, secrets)
-- [Auth0 Account](https://auth0.com/) (user management)
+- [jabba](https://github.com/shyiko/jabba) for managing Java versions
+- [nvm](https://github.com/nvm-sh/nvm) for managing Node versions
```

Comment:
Seems like `sbt` is also a requirement.

Reply:
Actually, that may have been a red herring. I think maybe the actual thing I missed is that `scripts/update` needs to be run from within the `app-backend` directory. Testing that out now, but if it's true, we probably want to modify `scripts/update`, so it can be run from the root directory.

Reply:
Made it through provisioning with the following change:

```diff
--- a/scripts/update
+++ b/scripts/update
@@ -21,8 +21,10 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
         usage
     else
         echo "Building Scala assembly JARs"
-        sbt \
-            ";api/assembly;backsplash-server/assembly;batch/assembly;backsplash-export/assembly"
+        pushd "app-backend"
+            sbt \
+                ";api/assembly;backsplash-server/assembly;batch/assembly;backsplash-export/assembly"
+        popd
 
         echo "Running application database migrations"
         ./scripts/migrate migrate
```